control_panel_api/permissions.py

```diff
@@ -20,6 +20,26 @@ def has_permission(self, request, view):
         return is_superuser(request.user)
 
 
+class K8sPermissions(BasePermission):
+    """
+    User can operate only in his namespace (unless superuser)
+    """
+
+    def has_permission(self, request, view):
+        if not request.user:
+            return False
+
+        if is_superuser(request.user):
```

Comment:
Can you not compose via mixin `IsSuperuser`?

Reply:
Or specify on the view and add the `IsSuperuser` permission?  Rather than rewriting and adding multiple permission checks.